misago/socialauth/admin/__init__.py

```diff
@@ -16,22 +16,22 @@ def register_urlpatterns(self, urlpatterns):
         urlpatterns.namespace(r"^social-auth/", "socialauth", "settings")
         urlpatterns.patterns(
             "settings:socialauth",
-            url(r"^$", SocialAuthProvidersList.as_view(), name="index"),
-            url(
-                r"^edit/(?P<pk>(\w|-)+)/$",
+            path("", SocialAuthProvidersList.as_view(), name="index"),
+            re_path(
+                "edit/(?P<pk>(\w|-)+)/$",
                 EditSocialAuthProvider.as_view(),
                 name="edit",
             ),
-            url(
-                r"^down/(?P<pk>(\w|-)+)/$",
+            re_path(
+                "down/(?P<pk>(\w|-)+)/$",
```

Comment:
regexp suggests that it is slug but name pk that it is int as above. What type is it?
in new version `\w` is deprecated

Reply:
It's a slug containing name of social auth provider, eg. `google-oauth2`